templates/pro/activate.html

```diff
@@ -0,0 +1,77 @@
+                <span class="p-radio__label" id="activate-myself">Myself</span>
+                <input type="hidden" name="activate-name" value="{{ user_info.fullname }}">
+              </label>
+            </div>
+            <div class="col-4">
+              <label class="p-radio">
+                <input class="p-radio__input radio-organisation" type="radio" aria-labelledby="activate-organisation-input" name="activate-buy-for" value="organisation">
+                <span class="p-radio__label" id="activate-organisation">Organisation</span>
+              </label>
+              <div class="p-form__group js-organisation-name-container" style="display:none">
+                <label for="activate-organisation-name" class="p-form__label">Organisation name: </label>
+                <input class="u-no-margin--bottom" type="text" id="activate-organisation-name" name="activate-organisation-name" placeholder="Ex: Canonical" style="margin-top: .25rem;" aria-label="What?">
+              </div>
+            </div>
+          </div>
+          <button class="p-button--positive" type="submit" name="submit">Submit</button>
+        </form>
+      </section>
+    </div>
+  {% endif %}
+
+  <section class="p-strip">
+    <div class="u-fixed-width">
+      <h2>Activate your Ubuntu Pro subscription</h2>
+      <p>
+        You are currently signed in as {{ user_info.fullname }}. Make sure it is the account you will be using to manage your subscription. If not, <a href="https://login.ubuntu.com/+login">log in to a different account</a>.
+      </p>
+      <p>
+        Your product key should be in the confirmation email you received after buying your Ubuntu Pro Desktop subscription.
+      </p>
+      <p>The activation key is 23 characters long and starts with a K.</p>
+      <form id="activate-key-form">
+        <div class="p-form__group">
+          <label for="pro-activation-key" class="p-form__label">Enter your code</label>
+          <div class="p-form__control is-error" style="min-width: 15rem">
+            <input type="text" id="pro-activation-key" class="activate-input" name="key" autocomplete="off" placeholder="Ex: Kabcedfghij12345678910-" required="" aria-describedby="pro-activate-input">
+            <p class="p-form-validation__message" id="pro-activatio-validation__message"></p>
+          </div>
+        </div>
+        <button class="p-button--positive" type="submit" name="submit">Activate</button>
+      </form>
+    </div>
+    <div class="u-fixed-width">
+      <div class="p-notification--negative" id="notification" style="display: none">
+        <div class="p-notification__content">
+          <h5 class="p-notification__title">Something went wrong</h5>
+            <p class="p-notification__message" id="notification-message"></p>
+        </div>
+        <div class="p-notification__meta">
+          <div class="p-notification__actions">
+            <a class="p-notification__action p-button" href="/pro/dashboard">Go to dashboard</a>
```

Comment:
Why link to the dashboard here? There is nothing the user could do there that would help with an activation error is there?

Reply:
Oh yeah right. "Go to dashboard" is actually supposed to display only for the "already used activation key" error message. The condition seems to be removed when we were refactoring a few days ago.